pkg/setting/setting.go

```diff
@@ -1297,9 +1298,27 @@ func readServerSettings(iniFile *ini.File, cfg *Cfg) error {
 		return err
 	}
 
+	cfg.CDNPath = valueAsString(server, "cdn_path", "")
 	return nil
 }
 
+// Returns full CDN Path with BuildVersion and edition added
+func (cfg *Cfg) GetFullCDNPath(edition string) string {
+	if cfg.CDNPath != "" {
+		cdnUrl, err := url.Parse(cfg.CDNPath)
+		if err == nil {
```

Comment:
I think this should be treated as an error and either logged or failing to start since the intent of user has been to configure this and if that fails the user should be notified.

Reply:
good point